test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -214,6 +214,13 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	},
 	// --
 
+	// k8s.io/kubernetes/pkg/apis/networking/v1
+	gvr("networking.k8s.io", "v1", "networkpolicies"): {
+		stub:             `{"metadata": {"name": "np1"}, "spec": {"podSelector": {"matchLabels": {"e": "f"}}}}`,
+		expectedEtcdPath: "/registry/networkpolicies/etcdstoragepathtestnamespace/np1",
+	},
```

Comment:
this should have `expectedGVK: gvkP("extensions", "v1beta1", "NetworkPolicy"),` to make sure we're still storing in a format 1.6 masters can read, at least for one version. In 1.7, we can switch to store in `network.k8s.io/v1` and run a migrate.

@enj I'm surprised this test didn't complain that either this or the extensions/v1beta1 NetworkPolicy wasn't getting stored in the keyed version